datadog/tests/datadog_test.go

```diff
@@ -19,10 +20,109 @@ import (
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
-// this test gets DD keys from env vars, writes a config file
-// then it invokes the plugin and validates some basics
-// in the response
-func TestDDCostRetrieval(t *testing.T) {
+func TestDDCostRetrievalListCost(t *testing.T) {
+	// query for qty 2 of 1 hour windows
+	windowStart := time.Date(2024, 3, 8, 0, 0, 0, 0, time.UTC)
+	windowEnd := time.Date(2024, 3, 8, 2, 0, 0, 0, time.UTC)
+
+	response := getResponse(t, windowStart, windowEnd, time.Hour)
+
+	// confirm no errors in result
+	if len(response) == 0 {
+		t.Fatalf("empty response")
+	}
+	for _, resp := range response {
+		if len(resp.Errors) > 0 {
+			t.Fatalf("got errors in response: %v", resp.Errors)
+		}
+	}
+
+	result, err := json.MarshalIndent(response, "", "    ")
+	if err != nil {
+		t.Fatalf("error json-ing response: %v", err)
+	}
+
+	fmt.Println(string(result))
```

Comment:
Its generally better to use `t.Log`/`t.Logf` (https://pkg.go.dev/testing#T.Logf) so that this output only shows up on failure or in verbose mode.

Reply:
Ah, I did not intend to commit this. Regardless, I appreciate the feedback!